db/seeds.rb

```diff
@@ -1,31 +1,32 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
 
+def translite(str)
+  Russian::translit(str).downcase.gsub(/\W+/, '-')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.